frontend/src/bundles/studio/components/timeline/timeline.tsx

```diff
@@ -28,11 +30,35 @@ const Timeline: React.FC<Properties> = ({ playerRef }) => {
 
     const range = useAppSelector(({ studio }) => studio.range);
 
+    const onButtonClick = useCallback(
+        (type: RowType) => {
+            switch (type) {
+                case RowNames.SCENE: {
+                    dispatch(studioActions.addScene());
+                    dispatch(
+                        studioActions.setMenuActiveItem(MenuItems.AVATARS),
+                    );
+                    break;
+                }
+                case RowNames.SCRIPT: {
+                    dispatch(studioActions.addScript(NEW_SCRIPT_TEXT));
+                    dispatch(studioActions.setMenuActiveItem(MenuItems.SCRIPT));
+                    break;
+                }
+            }
+        },
+        [dispatch],
+    );
+
```

Comment:
please change the names of the functions in this component so they start with handle instead of on